unified_planning/io/python_writer.py

```diff
@@ -137,8 +142,10 @@ def __init__(self, problem: 'up.model.Problem'):
 
     def _write_problem_code(self, out: IO[str]):
 
-        converter = ConverterToPythonString(self.problem.env)
+        names_mapping: Dict[str, str] = {}
```

Comment:
In order to avoid weird renamings, we could initialize this map with the valid names:
```
for all names n in self.problem:
  if is_valid_python_name(n):
    names_mapping[n] = n
```